.github/workflows/eaa_kbc.yml

```diff
@@ -25,34 +25,47 @@ jobs:
+          sudo apt-get update;
+          sudo apt-get install -y autoconf gcc g++ make wget git cmake \
           libseccomp-dev binutils-dev libprotoc-dev protobuf-compiler \
           pkg-config libssl-dev openssl libtool file curl gnupg;
 
-          wget https://github.com/protobuf-c/protobuf-c/archive/v1.3.1.tar.gz &&\
-          tar -zxvf v1.3.1.tar.gz && cd protobuf-c-1.3.1 && ./autogen.sh && \
-          ./configure && make && make install
+          sudo wget https://github.com/protobuf-c/protobuf-c/archive/v1.3.1.tar.gz;
+          sudo tar -zxvf v1.3.1.tar.gz;
+          cd protobuf-c-1.3.1 && sudo ./autogen.sh;
+          sudo ./configure;
+          sudo make;
+          sudo make install;
 
-          wget https://download.01.org/intel-sgx/sgx-linux/2.14/as.ld.objdump.gold.r3.tar.gz && \
-          tar -zxvf as.ld.objdump.gold.r3.tar.gz && cp -rf external/toolset/ubuntu18.04/* /usr/local/bin/ && \
-          rm -rf external && rm -rf as.ld.objdump.gold.r3.tar.gz
-
-          wget https://download.01.org/intel-sgx/sgx-linux/2.14/distro/ubuntu18.04-server/sgx_linux_x64_sdk_2.14.100.2.bin;
-          chmod +x sgx_linux_x64_sdk_2.14.100.2.bin &&  echo -e 'no\n/opt/intel\n' | ./sgx_linux_x64_sdk_2.14.100.2.bin;
-          rm -rf sgx_linux_x64_sdk_2.14.100.2.bin;
+          sudo wget https://download.01.org/intel-sgx/sgx-linux/2.14/as.ld.objdump.gold.r3.tar.gz;
+          sudo tar -zxvf as.ld.objdump.gold.r3.tar.gz;
+          sudo cp -rf external/toolset/ubuntu18.04/* /usr/local/bin/ && \
+          sudo rm -rf external && sudo rm -rf as.ld.objdump.gold.r3.tar.gz
 
+          sudo wget https://download.01.org/intel-sgx/sgx-linux/2.14/distro/ubuntu18.04-server/sgx_linux_x64_sdk_2.14.100.2.bin;
+          sudo chmod +x sgx_linux_x64_sdk_2.14.100.2.bin;
+          echo -e 'no\n/opt/intel\n' | sudo ./sgx_linux_x64_sdk_2.14.100.2.bin;
+          sudo rm -rf sgx_linux_x64_sdk_2.14.100.2.bin;
           source /opt/intel/sgxsdk/environment;
-
-          echo 'deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu bionic main' | tee /etc/apt/sources.list.d/intel-sgx.list && \
-          wget -qO - https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key | apt-key add -;
+          echo 'deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu bionic main' | sudo tee /etc/apt/sources.list.d/intel-sgx.list;
+          sudo rm /var/lib/apt/lists/lock && sudo rm /var/lib/dpkg/lock;
+          sudo apt-get --allow-unauthenticated upgrade;
+          sudo wget -qO - https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key | sudo apt-key add -;
           
-          apt-get update -y && apt-get install -y libsgx-dcap-quote-verify libsgx-dcap-default-qpl libsgx-dcap-ql-dev libsgx-uae-service libsgx-dcap-quote-verify-dev;
+          sudo apt-get update -y;
+          sudo apt-get install -y libsgx-dcap-quote-verify libsgx-dcap-default-qpl libsgx-dcap-ql-dev libsgx-uae-service libsgx-dcap-quote-verify-dev;
```

Comment:
It would be clearer if you used backslashes to fold each package onto a separate line (and ensured the list is sorted alphabetically). Same comment for other packages on this PR.